### ISSUE TITLE
cannot resolve path to local --compiler

### ISSUE THREAD

**Issue by harrysarson** [mpizenberg/elm-test-rs#66]
We shell out to the elm compiler here. However, because we set the `cwd` for the process we are trying to resolve the path relative to our scratch directory rather than relative to the directory from which elm-test-rs was run.

https://github.com/mpizenberg/elm-test-rs/blob/3f6e0a841f825063f970d8c6a6833bd581e380a1/src/run.rs#L485-L506

---

Therefore:

```
node-test-runner/tests/fixtures on  master [$!?] via 🌳 v0.19.1 
❯ elm-test-rs tests/Passing/One.elm --compiler ~/repo/node-test-runner/tests/fixtures/dummy-bin/different-elm                                             [22:45:04]

elm-test-rs 1.0.0-alpha for elm 0.19.1
--------------------------------

Total time spent in the elm compiler: 0.12971547s

Running 1 tests. To reproduce these results later,
run elm-test-rs with --seed 159612914 and --fuzz 100


TEST RUN PASSED

Duration: 1 ms
Passed:   1
Failed:   0
Running duration (since Node.js start): 104 ms


node-test-runner/tests/fixtures on  master [$!?] via 🌳 v0.19.1 
❯ elm-test-rs tests/Passing/One.elm --compiler ./dummy-bin/different-elm                                                                                  [22:51:25]

elm-test-rs 1.0.0-alpha for elm 0.19.1
--------------------------------

Error: Failed to run ./dummy-bin/different-elm. Are you sure it's in your PATH? If you installed elm locally with npm, maybe try running:

    npx --no-install elm-test-rs

since npx adds your locally installed packages to your PATH

Caused by:
    No such file or directory (os error 2)

```

**Comment by mpizenberg** [comment]
What is the correct way to handle this you think?

**Comment by mpizenberg** [comment]
Maybe if there are directories in the path (`some/directory/before/elm`) we convert it to an absolute path. And if there is no directory (`elm_bis`), we keep the current behavior since it's more likely another name for the binary.

**Comment by mpizenberg** [comment]
Seems reasonable fix. I implemented that in ce2987c. Let me know if you think there is a better idea.
